src/Jumbotron.js

```diff
@@ -2,11 +2,21 @@ import React from 'react';
 import classNames from 'classnames';
 
 const Jumbotron = React.createClass({
+  propTypes: {
+    componentClass: React.PropTypes.node.isRequired
```

Comment:
In the past I've personally used:
```
PropTypes.oneOf([PropTypes.string, PropTypes.func])
```
It doesn't catch everything but it gets closer at least :/ the more robust solution might be to add something to the CustomProptypes util like this: https://github.com/jquense/react-widgets/blob/master/src/util/propTypes.js#L10

for now though its probably fine to just go with `any` unless you want to tackle a more complete solution.